lib/internal/Magento/Framework/Translate.php

```diff
@@ -29,6 +29,13 @@ class Translate implements \Magento\Framework\TranslateInterface
      */
     protected $_localeCode;
 
+    /**
+     * Theme
+     *
+     * @var string
+     */
+    protected $_theme;
```

Comment:
Please do not use underscores in variable names and use private instead of protected.